internal/printer/printer.go

```diff
@@ -35,13 +35,11 @@ type Printer interface {
 	PrintPackage(pkg *pkg.Pkg, leadingNewline bool)
 	Printf(format string, args ...interface{})
 	OptPrintf(opt *Options, format string, args ...interface{})
+	OutStream() io.Writer
```

Comment:
I think we need a way for users to fetch the ErrStream too here. An example is this:
https://github.com/GoogleContainerTools/kpt/blob/fc64a0a358302ec41b3184188334e57fec778b1d/internal/gitutil/gitutil.go#L107
I assume we are doing something similar when piping the output from docker commands to the console.

Reply:
Updated.